src/coreclr/vm/methodtablebuilder.cpp

```diff
@@ -115,7 +115,7 @@ MethodTableBuilder::CreateClass( Module *pModule,
     }
     else
     {
-        pEEClass = new (pAllocator->GetLowFrequencyHeap(), pamTracker) EEClass(sizeof(EEClass));
+        pEEClass = new (pAllocator->GetLowFrequencyHeap(), pamTracker) EEClass();
```

Comment:
There are a few comments about not instantiating an EEClass with `new`, looks like we can remove or update them?

Reply:
Eh, `EEClass` still doesn't allow for the normal boring new operator. For correct usage, you still need the more complex new expression here. We could remove it, and require placement new in into a LoaderHeap in all scenarios, which would also work, but I don't see a need to remove the weird new behavior at this time.